src/main/java/gestorreservasaulas/entidades/Aula.java

```diff
@@ -0,0 +1,29 @@
+package gestorreservasaulas.entidades;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+
+@Entity
+@Table(name = "aulas")
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+public class Aula {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String nombre;
+    private int piso;
+    private int capacidad;
+
+    @ManyToOne
+    @JoinColumn(name="id_bloque")
+    private Bloque bloque;
+
```

Comment:
Es necesario la relacion de one to many a reservas y a horarios

Reply:
Se resolvió la relación one to many a reservas y a horarios, revisar para hacer merge